pkg/api/helper/helpers.go

```diff
@@ -140,11 +140,11 @@ func IsStandardContainerResourceName(str string) bool {
 	return standardContainerResources.Has(str) || IsHugePageResourceName(api.ResourceName(str))
 }
 
-// IsExtendedResourceName returns true if the resource name is not in the
-// default namespace, or it has the opaque integer resource prefix.
-func IsExtendedResourceName(name api.ResourceName) bool {
+// IsGenericResourceName returns true if the resource name is not in the
+// default namespace, or it has the opaque integer or hugepages resource prefix.
```

Comment:
I don't like this refactoring.
Extended resources != Integer resources in the long run.
I suspect what you are looking for is `IsIntegerResource` which I'd recommend composing